gapic/src/main/com/google/gapid/perfetto/models/SliceTrack.java

```diff
@@ -423,8 +463,8 @@ public Node build(Map<Long, List<Builder>> byParent) {
         "where ts < %d and ts + dur >= %d and depth >= %d and depth <= %d";
 
     private final QueryEngine qe;
-    private final String table;
-    private final long trackId;
+    protected final String table;
+    protected final long trackId;
```

Comment:
Let's reorder these to put `protected` and `private` together

Reply:
I think its better to have all the queries together for a neater look. But I can reorder them if needed.

Reply:
Yes, let's not impose arbitrary ordering.